pkgs/applications/misc/1password/default.nix

```diff
@@ -32,25 +42,27 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    install -D op $out/bin/op
+    install -D ${mainProgram} $out/bin/${mainProgram}
+    runHook postInstall
```

Comment:
Please also call preInstall

Reply:
Will do next time I touch this package.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/166339.